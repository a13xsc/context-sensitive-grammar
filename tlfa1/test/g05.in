({A,U,Y,_,1,N,b,6,M,2,h,B,/,W,E},{/,2,1,_,b,6,h},{(A,6MMb2BN//EYE/16/Y6N),(B,/WBM6),(B,EUEMEWA),(B,2WYY6M),(E,WU22BBAYh2Bh_EY),(M,NW6WEh6A/hYbbh6_B),(N,hWA_1W),(U,/YB6AhAM2b6E),(U,NA111/_B),(Y,/Yh6A2B266_BBN)},U)
