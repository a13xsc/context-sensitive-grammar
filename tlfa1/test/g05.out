N
W
Y
A
U
M
B
E
